src/Solver/Interoperability.cpp

```diff
@@ -469,57 +480,142 @@ void seissol::Interoperability::setupFSRMPointSources( double const* momentTenso
 void seissol::Interoperability::initializeModel(  char*   materialFileName,
                                                   int     anelasticity,
                                                   int     plasticity,
+                                                  int     anisotropy,
```

Comment:
I'd prefer to convert this to bool in the actual wrapper function; this would make the function below easier to read